module/VuFind/src/VuFind/RecordDriver/AbstractBase.php

```diff
@@ -154,8 +154,8 @@ abstract public function getUniqueID();
      */
     public function getComments()
     {
-        $table = $this->getDbTable('Comments');
-        return $table->getForResource(
+        $comments = $this->getDbService(\VuFind\Db\Service\CommentsServiceInterface::class);
```

Comment:
I hope to decouple comment retrieval from record drivers in the future, but I think it's best to finish this first incremental step before making more drastic changes.